framework/pages/bookstore/BookStorePage.ts

```diff
@@ -1,21 +1,33 @@
 import { Locator, Page } from "@playwright/test";
 import { BasePage } from "framework/pages/BasePage";
 import { BookSummaryFragment } from "./BookSummaryFragment";
+import { ChooseTheBook } from "../profile/ChooseTheBook";
 
 export class BookStorePage extends BasePage {
   url = "/books";
 
   //locators
   readonly logOutBtn: Locator = this.page.getByRole("button", { name: "Log out" });
   readonly goToBookStoreBtn: Locator = this.page.locator("id=gotoStore");
+  readonly bookTitles: Locator = this.page.locator("//span[@class='mr-2']")
+  readonly textSearchInput: Locator = this.page.locator("//input[@id='searchBox']");
+  readonly chooseTheBook: Locator = this.page.locator("//a[@href='/books?book=9781449331818']");
```

Comment:
this is not good to hardcode id's like 9781449331818
locator should be re-usable. Tomorrow book with id 9781449331818 may not exist